kubernetes/chart/zulip/templates/_helpers.tpl

```diff
@@ -65,16 +65,17 @@ Create the name of the service account to use
 include all env variables for Zulip pods
 */}}
 {{- define "zulip.env" -}}
+
 - name: DB_HOST
-  value: "{{ template "postgresql.primary.fullname" .Subcharts.postgresql }}"
+  value: "{{ template "postgresql.v1.primary.fullname" .Subcharts.postgresql }}"
 - name: DB_HOST_PORT
-  value: "{{ template "postgresql.service.port" .Subcharts.postgresql }}"
+  value: "{{ template "postgresql.v1.service.port" .Subcharts.postgresql }}"
 - name: DB_USER
   value: "postgres"
 - name: SETTING_MEMCACHED_LOCATION
   value: "{{ template "common.names.fullname" .Subcharts.memcached }}:11211"
 - name: SETTING_RABBITMQ_HOST
-  value: "{{ template "rabbitmq.fullname" .Subcharts.rabbitmq }}"
+  value: "{{ template "common.names.fullname" .Subcharts.rabbitmq }}"
```

Comment:
Can you point to documentation or some other way to check that these changes to the template are correct?

Reply:
i am not really shure when this was changed - but as you can see from fullname from memcached, the change matches to the memcached value for fullname, you can also see this in the _helpers.tpl in the bitnami repo: https://github.com/bitnami/charts/blob/main/bitnami/rabbitmq/templates/_helpers.tpl#L34

Reply:
WIth the suggested change `SETTING_RABBITMQ_HOST` get the value of `zulip-rabbitmq` (with a helm install name set to zulip), and the service name `zulip-rabbitmq`